Makefile.am

```diff
@@ -80,7 +80,8 @@ include_cpputest_HEADERS = \
 	include/CppUTest/TestResult.h \
 	include/CppUTest/TestTestingFixture.h \
 	include/CppUTest/Utest.h \
-	include/CppUTest/UtestMacros.h
+	include/CppUTest/UtestMacros.h \
+	CppUTestGeneratedConfig.h
```

Comment:
Recommend putting this into `include/CppUTest` as well.  Certainly it should be in that relative location when installed.

Also there should be a default file for VS and cmake builds.  Including regenerating it if autotools overwrote it, then someone does a subsequent cmake build.